src/modals/remove-liquidity-modal/remove-liquidity-modal.tsx

```diff
@@ -52,12 +53,12 @@ export const RemoveLiquidityModal = () => {
 
   const { publicKeyHash } = useSelectedAccountSelector();
   const styles = useRemoveLiquidityModalStyles();
-  const lpTokenMetadata = useTokenMetadataSelector(LIQUIDITY_BAKING_LP_SLUG);
 
-  const lpToken: TokenInterface = {
-    ...emptyToken,
-    ...lpTokenMetadata
-  };
+  const token = useMemo(
+    () =>
+      tokensList.find(candidateToken => getTokenSlug(candidateToken) === LIQUIDITY_BAKING_LP_SLUG) ?? { ...emptyToken },
```

Comment:
why do you make copy of `emptyToken`?

Reply:
because there can be no lp token

Reply:
return as it was, but without destructuring `emptyToken`